examples/custom-inspector/src/main.rs

```diff
@@ -0,0 +1,138 @@
+//! Example of how to use a custom inspector to trace new pending transactions
+//!
+//! Run with
+//!
+//! ```not_rust
+//! cargo run --release -p custom-inspector --node --http --ws --recipients 0x....,0x....
+//! ```
+//!
+//! If no recipients are specified, all transactions will be inspected.
+
+use clap::Parser;
+use futures_util::StreamExt;
+use reth::{
+    cli::{
+        components::{RethNodeComponents, RethRpcComponents, RethRpcServerHandles},
+        config::RethRpcConfig,
+        ext::{RethCliExt, RethNodeCommandConfig},
+        Cli,
+    },
+    primitives::{Address, BlockId, IntoRecoveredTransaction},
+    revm::{
+        inspector_handle_register, interpreter::Interpreter, revm, Database, EvmContext, Inspector,
+    },
+    rpc::{
+        compat::transaction::transaction_to_call_request,
+        eth::{revm_utils::EvmOverrides, EthTransactions},
+    },
+    tasks::TaskSpawner,
+    transaction_pool::TransactionPool,
+};
+use reth_revm::interpreter::OpCode;
```

Comment:
we should make this way less verbose. Prelude or otherwise?

Reply:
these are already the reexports -.-

but can def make this nicer